docs/user/04-metrics.md

```diff
@@ -349,11 +349,27 @@ spec:
 
 
 ### 4. Default Metrics for Telemetry Health
-By default, the metric gateway has the [kymastats receiver](https://github.com/kyma-project/opentelemetry-collector-components/blob/main/receiver/kymastatsreceiver/README.md) enabled. This receiver collects [metrics](https://github.com/kyma-project/opentelemetry-collector-components/blob/main/receiver/kymastatsreceiver/documentation.md) to monitor the health of following resources if they have been deployed:
-- Telemetry
-- Log Pipeline
-- Trace Pipeline
-- Metric Pipeline
+By default, a MetricPipeline emits metrics about the health of all pipelines managed by the Telemetry module. Based on these metrics, you can track the status of every individual pipeline and set up alerting for it. The following metrics are emitted for every pipeline and the Telemetry module itself:
+
+| metric                          | description                                                              | availability                                       |
+|---------------------------------|--------------------------------------------------------------------------|----------------------------------------------------|
+| kyma.resource.status.conditions | value represents status of different conditions reported by the resource | available for both pipeline and telemetry resource |
+| kyma.resource.status.state      | value represents the state of the resource (if present)                  | available for telemetry resource                   |
+
+The following metric attributes are available for the metric that could be used for monitoring:
+
+| Name                     | names                                                                                               |
+|--------------------------|-----------------------------------------------------------------------------------------------------|
+| metric.attributes.Type   | Type of the condition                                                                               |
+| metric.attributes.status | status of the condition                                                                             |
+| metric.attributes.reason | reason contains a programmatic identifier indicating the reason for the condition's last transition |
+
+
+A typical alert rule for looks like the following example. This alert would be triggered if metrics are not delivered to the backend for 5 minutes:
```

Comment:
- between "for" and "looks", a word is missing. If there's no word to add, remove the "for".
- Why "would be", why not "is"?

```suggestion
A typical alert rule for [ADD WORD] looks like the following example. The alert is triggered if metrics are not delivered to the backend for 5 minutes:
```